### ISSUE TITLE
which: invalid option -- 's'

### ISSUE THREAD

**Issue by W1zzardTPU** [pornel/pngquant#179]
```
[root@5654dc7e2b6e pornel-pngquant-6827a99]# ./configure

  Compiler: gcc
     Debug: no
       SSE: yes
    OpenMP: no
which: invalid option -- 's'
/usr/bin/libpng-config
    libpng: shared (1.6.16)
      zlib: shared (1.2.8)
     lcms2: no

[root@5654dc7e2b6e pornel-pngquant-6827a99]# uname -a
Linux 5654dc7e2b6e 3.10.0-229.14.1.el7.x86_64 #1 SMP Tue Sep 15 15:05:51 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux
[root@5654dc7e2b6e pornel-pngquant-6827a99]# cat /etc/redhat-release
Fedora release 22 (Twenty Two)
[root@5654dc7e2b6e pornel-pngquant-6827a99]# which -v
GNU which v2.20, Copyright (C) 1999 - 2008 Carlo Wood.
GNU which comes with ABSOLUTELY NO WARRANTY;
This program is free software; your freedom to use, change
and distribute this program is protected by the GPL.

```

**Comment by pornel** [comment]
What is your OS?

Please run:

```
bash -x ./configure
```

**Comment by sergiomb2** [comment]
I got this also is: 
> \+ which -s libpng-config
> which: invalid option -- 's'
> /usr/bin/libpng-config

**Comment by pornel** [comment]
Also please run 

    bash -x ./configure

and share the log so I can debug the problem

**Comment by sergiomb2** [comment]
I sent the result of bash -x ./configure and it easy to find, line 312 of configure: 

    if which -s libpng-config; then

**Comment by pornel** [comment]
Thanks, I'll fix it in the next release

**Comment by pornel** [comment]
Fixed in 308afdc5534c0a85ac36c76b8f91723bf3f32349
